frontend/src/metabase/query_builder/components/filters/modals/InlineValuePicker/constants.ts

```diff
@@ -0,0 +1 @@
+export const ALLOWED_OPERATORS = ["=", "!="];
```

Comment:
This doesn't seem to be used anymore? It's imported in `InlineValuePicker.tsx` but also isn't used there.